bazel/repository_locations.bzl

```diff
@@ -10,10 +10,10 @@ REPOSITORY_LOCATIONS = dict(
     ),
     boringssl = dict(
         # Use commits from branch "chromium-stable-with-bazel"
```

Comment:
This is no longer stable branch. Can you update comment?

Reply:
Done.